File: docs/components_page/components/badge/links.py
import dash_bootstrap_components as dbc
from dash import html

badges = html.Span(
    [
        dbc.Badge(
            "Primary",
            href="#",
            color="primary",
            className="me-1 text-decoration-none",
        ),
        dbc.Badge(
            "Secondary",
            href="#",
            color="secondary",
            className="me-1 text-decoration-none",
        ),
        dbc.Badge(
            "Success",
            href="#",
            color="success",
            className="me-1 text-decoration-none",
        ),
        dbc.Badge(
            "Warning",
            href="#",
            color="warning",
            className="me-1 text-decoration-none",
        ),
        dbc.Badge(
            "Danger",
            href="#",
            color="danger",
            className="me-1 text-decoration-none",
        ),
        dbc.Badge(
            "Info",
            href="#",
            color="info",
            className="me-1 text-decoration-none",
        ),
        dbc.Badge(
            "Light",
            href="#",
            color="light",
            text_color="dark",
            className="me-1 text-decoration-none",
        ),
        dbc.Badge("Dark", href="#", color="dark", className="text-decoration-none"),
    ]
)

File: components_page/components/badge/links.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

badges <- htmlSpan(
  list(
    dbcBadge("Primary", href = "#", color = "primary", class_name = "me-1"),
    dbcBadge("Secondary", href = "#", color = "secondary", class_name = "me-1"),
    dbcBadge("Success", href = "#", color = "success", class_name = "me-1"),
    dbcBadge("Warning", href = "#", color = "warning", class_name = "me-1"),
    dbcBadge("Danger", href = "#", color = "danger", class_name = "me-1"),
    dbcBadge("Info", href = "#", color = "info", class_name = "me-1"),
    dbcBadge("Light", href = "#", color = "light", class_name = "me-1"),
    dbcBadge("Dark", href = "#", color = "dark")
  )
)
